src/test/java/org/rulez/demokracia/PDEngine/VoteCreationTest.java

```diff
@@ -110,4 +111,340 @@ public void create_creates_a_vote_with_adminKey() {
 		assertEquals(VoteRegistry.getByKey(adminInfo.adminKey).adminKey, adminInfo.adminKey);
 	}
 
+// voteName validation tests
+
+	@Test
+	public void votename_contains_wrong_characters_input1() throws ReportedException {
+		List<String> neededAssurances = new ArrayList<>();
+		List<String> countedAssurances= new ArrayList<>();
+	boolean isPrivate = true;
+		int minEndorsements = 0;
+		neededAssurances.add("magyar");
+		String wrongVotename = "This contains spaces";
+
+		try {
+			VoteAdminInfo secondVote = VoteRegistry.create(wrongVotename,neededAssurances, countedAssurances, isPrivate, minEndorsements );
+			fail();
+		} catch (ReportedException e) {
+			// expected
+			assertEquals(e.getMessage(),"Wrong characters in the vote name!");
+		}
+
+	}
+
+	@Test
+	public void votename_contains_wrong_characters_input2() {
+		List<String> neededAssurances = new ArrayList<>();
+		List<String> countedAssurances= new ArrayList<>();
+		boolean isPrivate = true;
+		int minEndorsements = 0;
+		neededAssurances.add("magyar");
```

Comment:
what about creating a function which needs only the vote name, initializes other parameters and calls VoteRegistry.create
you could use it in all tests, making a more compact code.